### DISCUSSION TITLE
Reelz (US)

### DISCUSSION THREAD

**Discussion by industrystandard1** [iptv-org/community#547] [Q&A] [answered]
https://bcovlive-a.akamaihd.net/c733a9aa448a4a44a10c527c6f5bf7a4/us-east-1/5245389775001/playlist.m3u8

went offline a few months ago. Multiple users have posted about it, still has not been fixed:

https://github.com/iptv-org/iptv/issues/15260

I made a post about it here:

https://github.com/iptv-org/iptv/issues/15508

Was told to post about it in the discussions forum. Is this going to be updated or are we screwed on finding a Reelz stream? I know certain paid-for (monetized) IPTV products on jailbroken firesticks have a direct line to Reelz still functioning.

**Comment by industrystandard1** [comment]
Bump

**Comment by BellezaEmporium** [comment]
Bump @R4IN2007 (s)he usually finds out the bcov links for those kind of channels.

**Comment by industrystandard1** [comment]
bump, my favorite show comes on tomorrow :/ have missed it for the past few months since this stream went offline

**Comment by industrystandard1** [comment, marked as answer]
Holy sh*t it's back online, whoever saw this thread and fixed it bless you.

https://iptv-org.github.io/?q=reelz

https://bcovlive-a.akamaihd.net/rc21a8bc6455344c699a06e9220fe2748/us-west-2/5245389775001/playlist.m3u8

**Reply by R4IN2007** [reply]
https://github.com/iptv-org/iptv/issues/15685
